### PR TITLE
Merged FE

### PR DESCRIPTION
**Pull request by surajpkumar08**
1. Merged Tagging,Pagination and CSS with existing FE
2. Removed Warnings
3. Loading
4. Separation Of Concerns
5. Linkage in Contribution

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -10,6 +10,14 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
```
Comment (mayank0409) [inline public/index.html:15]: Why are we using bootstrap here?

File: src/Components/Box/index.js [hunk 2]
```diff
@@ -7,14 +7,33 @@ import Contribution from '../Contribution';
 
 export default class Box extends Component {
 
+  clickEvent()
+  {
+
+  }
```
Comment (mayank0409) [inline src/Components/Box/index.js]: empty function ?

File: public/index.html (hunk 1 above)
Comment (surajpkumar08) [reply public/index.html:15]: Sir, for 1 alignment and tags we are using bootstrap

File: src/Components/Box/index.js (hunk 2 above)
Comment (surajpkumar08) [reply src/Components/Box/index.js]: This function is required for on click in iphone

File: src/Components/Search/index.test.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    // });
+
+   
+
+    // const spy = expect.spyOn(Search.prototype, "onButtonClick");
+    // const wrapper = mount(<Search />);
+    // expect(spy).toNotHaveBeenCalled();
+    // wrapper.find("Go").simulate("click");
+    // expect(spy).toHaveBeenCalled();
+    // spy.restore();
```
Comment (mayank0409) [inline src/Components/Search/index.test.js]: Please remove dead/commented code

**Pull request merged**